File: fullConverterwithMessage.py
def changeMoney(dollars):
    francs = dollars*1.02
    message = dollars+" dollars are "+francs+" francs."
    return message

print(changeMoney(25))

def changeWater(liters):
    gallons = liters*0.17
    message = liters+" liters are "+gallons+" gallons."
    return message

print(changeWater(3))

def changeHeigth(meters):
    centimeters = meters/100
    message = centimeters+" centimeters are "+meters+" meters."
    return message

print(changeHeigth(34))

File: fullConverterwithMessage.js
function changeMoney(dollars){
    var francs = dollars*1.02;
    var message = dollars+" dollars are "+francs+" francs.";
    return message;
}
console.log(changeMoney(25));

function changeWater(liters){
    var gallons = liters*0.17;
    var message = liters+" liters are "+gallons+" gallons.";
    return message;
}
console.log(changeWater(3));

function changeHeigth(meters){
    var centimeters = meters*100;
    var message = centimeters+" centimeters are "+meters+" meters."
    return message
}
console.log(changeHeigth(34));